tests/git_bisect_toaster.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# These variables are fully defined in runtests.sh
+# Just short descripstions appear here
+# what container to run. Usually crops/toaster-master
+export IMAGE=crops/toaster-master
```

Comment:
Oh right, and since we can't use virtualenv, we can't install multiple versions.